app/controllers/schools/advice/electricity_recent_changes_controller.rb

```diff
@@ -2,6 +2,10 @@ module Schools
   module Advice
     class ElectricityRecentChangesController < AdviceBaseController
       def insights
+        @recent_usage = ::Usage::RecentUsageComparisonService.new(
```

Comment:
As noted on Trello, suggest we revise this service to allow it to return comparison for last 2 school weeks or last 4 weeks.